commands/slash/247.js

```diff
@@ -58,14 +58,16 @@ const command = new SlashCommand()
       }`
     );
 
-    if (!player.playing && player.queue.totalSize === 0) player.destroy();
+    if (!player.playing && player.queue.totalSize === 0 && twentyFourSeven) player.destroy();
```

Comment:
This will do exactly the opposite you wanna do. Destroying the player if 24/7 is enabled

Reply:
Not really. When 247 is enabled and the command is run, so the user actually requests to disable the 247 so the 247 turns off and the player destroys.
Here's a preview of what 247 turned on and off:
![image](https://user-images.githubusercontent.com/84950599/174042200-55df8c88-607d-4f7e-bd4a-1b8f12858ff1.png)

![image](https://user-images.githubusercontent.com/84950599/174041938-67b75b99-4f98-4111-bbc0-ff2787da22c6.png)

Reply:
I agree with @Neko-Life , the logic doesn't work here:
It's saying "Destroy the player if the player is not playing, there are no songs in queue and 24/7 is active"
Which is just the problem you described. the correct logic would be
```js
if (!player.playing && player.queue.totalSize === 0 && !twentyFourSeven) player.destroy();
```

Reply:
> > ![image](https://user-images.githubusercontent.com/84950599/174041938-67b75b99-4f98-4111-bbc0-ff2787da22c6.png)

This is the intended behavior

Reply:
The original issue was if the bot is connected to the channel with `/summon` and then `/247` was run it would just disconnect. So I thought, this might be a fix so that if 247 is made `DISABLED -> ENABLED` then it wouldn't disconnect, it would just turn 24/7 on. If it is made `ENABLED -> DISABLED` then the bot will say **24/7 is OFF** and disconnect from the vc.

Reply:
> The original issue was if the bot is connected to the channel with `/summon` and then `/247` was run it would just disconnect. So I thought, this might be a fix so that if 247 is made `DISABLED -> ENABLED` then it wouldn't disconnect, it would just turn 24/7 on. If it is made `ENABLED -> DISABLED` then the bot will say **24/7 is OFF** and disconnect from the vc.

That was exactly what  I was trying to do, thanks for finding this bug

Reply:
I just opened the command file and yes, the code was written to be not easily read nor understand. I approve this PR.